GVFS/GVFS.Common/InstallerPreRunChecker.cs

```diff
@@ -202,7 +202,9 @@ private bool IsGVFSUpgradeAllowed(out string consoleError)
 
             if (this.IsServiceInstalledAndNotRunning())
             {
-                adviceText = isConfirmed ? $"Run `sc start GVFS.Service` and run {this.CommandToRerun} again from an elevated command prompt." : $"To install, run `sc start GVFS.Service` and run {GVFSConstants.UpgradeVerbMessages.GVFSUpgradeConfirm} from an elevated command prompt.";
+                adviceText = isConfirmed
+                    ? $"Run {GVFSPlatform.Instance.Constants.StartServiceCommandMessage} and run {this.CommandToRerun} again."

```

Comment:
To we need two different messages here?

if `isConfirmed` is true, then `CommandToRerun` is the same as `UpgradeConfirmCommandMessage` and so the difference between messages is minimal:

```
 $"Run {GVFSPlatform.Instance.Constants.StartServiceCommandMessage} and run {UpgradeConfirmCommandMessage} again."
```
vs
```
$"To install, run {GVFSPlatform.Instance.Constants.StartServiceCommandMessage} and run {GVFSPlatform.Instance.Constants.UpgradeConfirmCommandMessage}."
```